### DISCUSSION TITLE
请问如何进入后台正在运行的nonebot的命令行

### DISCUSSION THREAD

**Discussion by zlqd123** [nonebot/discussions#547] [问答] [answered]
使用nonebot-bison的docker运行，在某一次意外关机后，启动docker会自动执行nb run，无法再手动执行，导致无法在看到nb的输出，nb是在正常工作的。
想问一下怎么进入后台正在运行的nonebot的命令行界面

**Comment by yanyongyu** [comment, marked as answer]
没看懂，你的 nb 是在 docker 内运行的？那你直接 `docker logs <container-name>` 就可以看到日志吧。

**Reply by zlqd123** [reply]
> docker logs

用这个方法确实能看到输出，谢谢。
在此基础上我又查了一下，使用 `docker logs -f -t <container-name>` 就可以实时输出带时间戳的日志了。

是用的这个docker， https://hub.docker.com/r/felinae98/nonebot-bison 
包含了nb 和bison插件。
